amanuensis/resources/userdatamodel/userdatamodel_notification.py

```diff
@@ -0,0 +1,110 @@
+from flask import current_app
+from sqlalchemy import func
+from cdislogging import get_logger
+from amanuensis.errors import NotFound, UserError
+from amanuensis.models import (
+    Notification, NotificationLog
+)
+
+# do this until AWS-related requests is handled by it's own project
+from pcdcutils.environment import is_env_enabled
+
+logger = get_logger(__name__)
+
+__all__ = [
+    "get_all_notification",
+    "get_all_notiflogs",
+    "get_notifications_by_user",
+    "get_log_associated",
+    "get_notification_by_log",
+    "check_seen"
+    "check_unseen_notifications",
+    "create_new_notification",
+    "change_seen_status",
+    "delete_object"
+]
+
+# Get all notifications in the DB
+def get_all_notifications(current_session):
+    notifications = current_session.query(Notification)
+    return notifications
+
+
+def get_all_notiflogs(current_session):
+    logs = current_session.query(NotificationLog)
+    return logs
+
+
+# Get all notifications associated with a user
+def get_notifications_by_user(current_session, logged_user_id):
+    notifications = current_session.query(Notification).filter_by(user_id = logged_user_id)
+    return notifications
+
+
+# Get all notifications associated with a log
+def get_log_assoiciated(current_session, log_id):
```

Comment:
both this functions should have a seen parameter defaulted to null. If the parameter is not null then you should append to the query WHERE seen = ...